twolde.py

```diff
@@ -205,11 +207,18 @@ def nice_time(seconds):
 
 def main():
     """ main """
-    parser = argparse.ArgumentParser(description='')
+    parser = argparse.ArgumentParser(
+        description="It's like Twitshift, but for people who prefer Twolde",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
     parser.add_argument(
         dest='command',
         choices=['install', 'run', 'remove'],
         help='the twolde command to run')
+    parser.add_argument(
```

Comment:
The years option only really makes sense for the run command. Would be nice to use `add_subparsers` instead of the choices and add the years option to the run subparser. This is how I originally implemented it, but choices seemed neater when there were no sub options.